reactor-core/src/test/java/reactor/core/publisher/FluxReplayTest.java

```diff
@@ -383,7 +461,7 @@ public void minimalInitialRequestWithUnboundedSubscriber() {
 		replay.subscribe(); //unbounded
 		replay.connect();
 
-		assertThat(requests).containsExactly(Long.MAX_VALUE);
+		assertThat(requests).containsExactly(3L, 3L);
```

Comment:
these tests need to be a bit more demonstrative. maybe increase the range, so that we see that here it makes enough replenishing requests to come to completion, due to unbounded subscriber ?

Reply:
fixed